CHANGELOG.md

```diff
@@ -24,6 +24,28 @@ CHANGELOG
 Swift 5.0
 ---------
 
+* [SE-0235][]:
+
+  The standard library now contains a `Result<Success, Failure: Error>` type to 
+  manually model potentially error-producing code.
+  
+  ```swift
+  enum Result<Success, Failure: Error> {
+      case success(Success)
+      case failure(Failure)
+  }
+  ```
+  
+  This type serves a separate but complementary role to Swift's existing and 
+  automatic error handling using `try` / `catch`. In addition to it's usefulness 
+  in asynchronous code, `Result` can be used in situations where the formality of
```

Comment:
```suggestion
  `Result` can be used where implicit error propagation and the ceremony of
```

"In situations where" is redundant.  
Specific examples can come afterwards instead of sandwiching the explanation of the examples.  
"Formality" has ambiguous connotations.

Reply:
I swapped "automatic" for "implicit" in your suggested change, as I don't think the use of explicit keywords is very implicit, and I've seen "automatic" more commonly used to describe Swift's error handling, especially in the error manifesto.